wezterm/wezterm.lua

```
@@ -0,0 +1,232 @@
+    -- ------------------------------------------------------------------------
+    -- ペインを閉じる
+    -- ------------------------------------------------------------------------
+    { key = 'w', mods = 'CMD|SHIFT', action = wezterm.action.CloseCurrentPane{ confirm = true } },  -- Cmd+Shift+w: 現在のペインを閉じる(確認あり)
+}
+
+-- ============================================================================
+-- イベントハンドラー
+-- ============================================================================
+
+-- 設定ファイルがリロードされた時にログに通知を記録
+wezterm.on('window-config-reloaded', function(window, pane)
```

Luacheck (1.2.0)

[warning] unused argument 'window'

(W212)